CDP4CrossViewEditor/ViewModels/ParameterTypeSelectorViewModel.cs

```diff
@@ -46,7 +46,7 @@ public sealed class ParameterTypeSelectorViewModel : ThingSelectorViewModel
         /// <summary>
         /// Hardcoded power related parameter type short names list
         /// </summary>
-        private static readonly string[] PowerParameters = { "redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
+        private static readonly string[] PowerParameters = { "Redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
```

Comment:
We might want to make the comparison case insensitive where these are checked (if not already done).

Reply:
Correct.